redfish/resource_redfish_dell_idrac_attributes.go

```diff
@@ -231,7 +231,7 @@ func getManagerAttributeRegistry(service *gofish.Service) (*dell.ManagerAttribut
 	for _, r := range registries {
 		if r.ID == "ManagerAttributeRegistry" {
 			// Get ManagerAttributesRegistry
-			managerAttrRegistry, err := dell.GetDellManagerAttributeRegistry(service.Client, r.Location[0].URI)
+			managerAttrRegistry, err := dell.GetDellManagerAttributeRegistry(service.GetClient(), r.Location[0].URI)
```

Comment:
Are we sure r.Location[0].URI will not result in nil reference?

Reply:
GetClient() changes are to adopt latest gofish version. These changes are not related to virtual media resource implementation.